language/examples/document-summarization/summarization_with_docai.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>De-emphasize the financial report documents by changing the first sentence to:</p><p> </p><p>In this notebook, you learned: </p><ol><li>...</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Done</p>